test-apps/ui-test-app/src/frontend/appui/frontstages/IModelIndexFrontstage.tsx

```diff
@@ -55,6 +55,7 @@ export class IModelIndexFrontstage extends FrontstageProvider {
         defaultTool={CoreTools.selectElementCommand}
         contentGroup={contentGroup}
         isInFooterMode={false}
+        isModelIndependent={true}
```

Comment:
Is this needed because the UiFramework holds onto the current iModel? So these components need to be re-render when the current iModel changes?

Reply:
good point... will remove isModelIndependent

Reply:
fixed - this is needed because of the way the static FrontstageManger caches frontstageDef... these typically should be cached by active imodel since there is stale imodel specific data in the def.... at some point in the future this should probably be rearchitected to work with real React component instead of the def but that is a big project.

Reply:
okay, sounds good.

(Definitely agree on the re-architecture needed at some point...)